build-tools/bin/run_tests.sh

```diff
@@ -196,8 +237,44 @@ if [[ -z "${SKIP_SECURITY_MANAGER}" ]]; then
 fi
 
 if [[ -z "${SKIP_STATIC}" ]]; then
+    echo
+    echo "******************"
+    echo "Running AgentMode Verbose Tests"
+    echo "------------------"
+
+    function test_static_agent_verbose() {
+      static_agent_configure_verbose ${JDK_DIR} ${AGENT_JAR} $1 $2
```

Comment:
Add comments/docs on what the values mean. digging through the code is not maintainable.